ScarletEditor/Sources/SceneHierarchy/Private/SceneHierarchy.cpp

```diff
@@ -30,12 +28,12 @@ namespace ScarletEngine
 	void SceneHierarchyPanel::DrawWindowContent()
 	{
 		ZoneScoped
-		ImGuiTreeNodeFlags BaseFlags = ImGuiTreeNodeFlags_OpenOnArrow | ImGuiTreeNodeFlags_OpenOnDoubleClick | ImGuiTreeNodeFlags_SpanAvailWidth;
+		const ImGuiTreeNodeFlags BaseFlags = ImGuiTreeNodeFlags_OpenOnArrow | ImGuiTreeNodeFlags_OpenOnDoubleClick | ImGuiTreeNodeFlags_SpanAvailWidth;
 		for (const auto& [ID, EntItem] : Items)
 		{
-			ImGui::PushID((uint32_t)ID);
+			ImGui::PushID(static_cast<uint32_t>(ID));
 			char Buffer[128];
-			snprintf(Buffer, 128, "%s %s", ICON_MD_CUBE, EntItem->GetDisplayString());
+			snprintf(Buffer, 128, "%s   %s", ICON_MD_CUBE, EntItem->GetDisplayString());
```

Comment:
why extra space?

Reply:
Makes the scene hierarchy a little less compressed, adds some space between the icon and the label.